### PR TITLE
Update af.m3u

### PR DESCRIPTION
**Pull request by trancer1994**

### CODE REVIEW

File: channels/af.m3u [hunk 1]
```diff
@@ -9,3 +9,39 @@ https://d2g7v53450s2i2.cloudfront.net/ATNUS/myStream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="Hewad TV" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/RkcUtMK.jpg" group-title="",Hewad TV
+http://51.210.199.58/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Noor Dubai
+http://dmiffthvll.cdn.mangomolo.com/noordubaitv/smil:noordubaitv.smil/chunklist_b725000.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Noor Dubai
+http://www.elahmad.com/tv/m3u8/dubaitv.m3u8?id=noordubaitv
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",NOOR DUBAI HD
+http://dmiffthvll.cdn.mangomolo.com/noordubaitv/smil:noordubaitv.smil/chunklist_b1500000.m3u8?
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Noor Dubia
+http://dmiffthvll.cdn.mangomolo.com/noordubaitv/smil:noordubaitv.smil/chunklist_b1500000.m3u8?3572333505
```
Comment (freearhey) [inline channels/af.m3u:45]: Noor Dubia is a channel from the United Arab Emirates: https://www.lyngsat.com/tvchannels/ae/Noor-Dubai.html

File: channels/af.m3u [hunk 2]
```diff
@@ -9,3 +9,39 @@ https://d2g7v53450s2i2.cloudfront.net/ATNUS/myStream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="https://i.imgur.com/oeV42l0.jpg" group-title="",Noor TV
+https://ls1.serverdump.com/stream3.m3u8
```
Comment (freearhey) [inline channels/af.m3u:47]: This is UK channel: https://www.lyngsat.com/tvchannels/uk/Noor-TV.html

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/af.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/af.m3u:47]: Can confirm, also says it on their website : https://www.thenoor.tv/ "Noor Islamic TV Channel is the UK based Satellite Television Channel"

**Comment by freearhey** [comment]
Closed due to lack of any activity.

**Pull request closed without merging**